### PR TITLE
Add 2016 score breakdown

### PR DESCRIPTION
**Pull request by phil-lopreiato**
See http://tba.lopreiato.me/match/2016scmb_qm3 as an example

### CODE REVIEW

File: controllers/match_controller.py [hunk 1]
```diff
@@ -16,6 +16,17 @@ class MatchDetail(CacheableHandler):
     CACHE_VERSION = 4
     CACHE_KEY_FORMAT = "match_detail_{}"  # (match_key)
 
+    defense_render_names = {
```
Comment (fangeugene) [inline controllers/match_controller.py]: This year specific stuff (along with `add_defense_render_names`) is awkward in the default match controller... I'm trying to think of a better way to do this.

Comment (phil-lopreiato) [inline controllers/match_controller.py]: Yeah, not a huge fan of this. I should case the year, at least

Comment (fangeugene) [inline controllers/match_controller.py]: Oh, idea. Add a filter to `common/my_filters.py`.

Comment (fangeugene) [inline controllers/match_controller.py]: Call it like `render_defense_name` or something.

**Pull request merged**